### PR TITLE
Suggestions for Deck 6 Hyperparameter tuning

### PR DESCRIPTION
**Pull request by hfrick**
Just very minimal suggestions and one fix (`_wflow` instead of `_flow`)

### CODE REVIEW

File: slides/06-tuning-hyperparameters.qmd [hunk 1]
```diff
@@ -173,16 +173,16 @@ glm_wflow <-
 
 #### Parameters
 
--   The tidymodels framework provides pre-defined information on tuning parameters (such as their type, range, transformations, etc)
+-   The tidymodels framework provides pre-defined information on tuning parameters (such as their type, range, transformations, etc)
```
Comment (hfrick) [inline slides/06-tuning-hyperparameters.qmd:176]:
```suggestion
-   The tidymodels framework provides pre-defined information on tuning parameters (such as their type, range, transformations, etc) 
```
```suggestion
-   The tidymodels framework provides pre-defined information on tuning parameters (such as their type, range, transformations, etc).
```

File: slides/06-tuning-hyperparameters.qmd [hunk 2]
```diff
@@ -173,16 +173,16 @@ glm_wflow <-
...
+-   The `extract_parameter_set_dials()` function extracts these tuning parameters and the info 
 
 ::: fragment
 #### Grids
 
--   Create your grid manually or automatically
+-   Create your grid manually or automatically 
 
--   The `grid_*()` functions can make a grid
+-   The `grid_*()` functions can make a grid
```
Comment (hfrick) [inline slides/06-tuning-hyperparameters.qmd:185]:
```suggestion
-   The `grid_*()` functions can make a grid.
```

File: slides/06-tuning-hyperparameters.qmd [hunk 3]
```diff
@@ -198,7 +198,7 @@ glm_wflow %>%
 ```
 
 ::: fragment
-A parameter set can be updated (e.g. to change the ranges)
+A parameter set can be updated (e.g. to change the ranges)
```
Comment (hfrick) [inline slides/06-tuning-hyperparameters.qmd:201]:
```suggestion
A parameter set can be updated (e.g. to change the ranges).
```

File: slides/06-tuning-hyperparameters.qmd [hunk 4]
```diff
@@ -289,7 +289,7 @@ set.seed(9)
 glm_res <-
   glm_wflow %>%
   tune_grid(resamples = nhl_val, grid = grid, control = ctrl)
-glm_res
+glm_res
```
[the remark this answers is not in the document]
Comment (topepo) [reply slides/06-tuning-hyperparameters.qmd:292]: Yes, we should. Can you add some notes?

[the remark this answers is not in the document]
Comment (topepo) [reply slides/06-tuning-hyperparameters.qmd]: @hfrick I added details in the annotations (see d8a3130 and edit as needed)

@EmilHvitfeldt Can you diagnose why the annotation link doesn't work? There were hexes in the title for that slide (and it wasn't unique). I fixed those issues and it still does not show up 😢

[the remark this answers is not in the document]
Comment (EmilHvitfeldt) [reply slides/06-tuning-hyperparameters.qmd]: Did you add `include-after-body: footer-annotations.html` in the header?

**Pull request merged**